src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp

```diff
@@ -69,6 +70,17 @@ WellSolverBase::~WellSolverBase() = default;
 void WellSolverBase::postProcessInput()
 {
   SolverBase::postProcessInput();
+
+  // create dir for rates output
+  if( getLogLevel() > 0 )
+  {
+    if( MpiWrapper::commRank() == 0 )
+    {
+      makeDirsForPath( m_ratesOutputDir );
```

Comment:
Just for discussion not as a review note, as we are C++17 compliant, is there any plan/advantage to use `std::filesystem::path` or specific path structure @TotoGaz @klevzoff  ?

Reply:
We encourage using it for new code, but there is no concerted effort to convert all existing path-handling code